htdocs/export.bml.text

```diff
@@ -29,7 +29,7 @@
 
 .label.format=Format:
 
-.label.header=Header:
+.label.header=Include header line
```

Comment:
I like this text so much better... but I don't see it on the new version of the page!